### ISSUE TITLE
insights: Extend/change add/remove gql mutation in a way to handle list of insights per one mutation 

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#32194]
Related GH thread https://github.com/sourcegraph/sourcegraph/issues/23065#issuecomment-1057858501

## Background 

In the nutshell, at the moment `addInsightViewToDashboard` and `removeInsightViewFromDashboard` mutations take only one insight id. But it would be better from the performance point of view for the frontend to have mutations that could take a list of insights for adding or removing them to/from the dashboard. 

A good use case for such mutations might be our "Add or remove insights" modal view. At the moment we are sending a package (bulk) GQL mutation request for each added/removed insight there. With a new mutation, we could just run one mutation for adding and one for removing. 

![](https://user-images.githubusercontent.com/18492575/156624806-a31f6922-256b-4f4d-9f91-dbf78a01bbc1.png)

**Comment by vovakulikov** [comment]
@coury-clark @CristinaBirkel I think this might be a good first issue and if you don't mind I would like to try to implement this change by myself with your review of course.

**Comment by coury-clark** [comment]
@vovakulikov of course! Having said that, unless you change the underlying query pattern (to bulk insert / delete) I don't see any substantial difference from just sending multiple mutations in a single request. Is that something you've tried?

**Comment by vovakulikov** [comment]
@coury-clark 
> sending multiple mutations in a single request.

This is actually the current state for assigning/removing insight to/from dashboard
